flask_alchemydumps/helper/database.py

```diff
@@ -41,3 +42,8 @@ def parse_data(self, contents):
         """Loads a dump and convert it into rows """
         db = self.db()
         return loads(contents, db.metadata, db.session)
+
+    def get_next_sequence(self, sequence_name):
```

Comment:
@cuducos Added get_next_sequence method for getting sequence, so there wont be any id-mismatch happening while inserting anything new in table.